drafts/2020-02-20-Rest-client-generator.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "The OpenAPI Generator has added support for Eclipse MicroProfile"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/turkeylurkey
+seo-title: OpenAPI Generator has added support for Eclipse MicroProfile - OpenLiberty.io
+seo-description: Develop microservices quicker using the OpenAPI Generator and the Eclipse MicroProfile programming framework.
+blog_description: "Develop microservices quicker using the OpenAPI Generator and the Eclipse MicroProfile programming framework."
```

Comment:
Update these metadata titles/descriptions to be more aligned with the updated title below ("Generate MicroProfile REST client code to send requests to other microservices")

Reply:
Ok